src/tbb_concurrent_queue_scheduler.cpp

```diff
@@ -231,7 +233,7 @@ namespace CnC {
 					m_queues[_aff].push( stepInstance );
 				}
             } else {
-                m_gQueue->push( stepInstance );

```

Comment:
Why doesn't removing this push to m_gQueue cause problems with the blocking pop above?